can-i-deploy/canideployTo.sh

```diff
@@ -21,6 +21,14 @@ if [ ${#MISSING[@]} -gt 0 ]; then
   exit 1
 fi
 
+PACT_CLI_IMAGE=
+if [ "$pact_cli_image" ]; then
+    echo "You set pact cli image"
```

Comment:
```suggestion
    echo "INFO: using user-specified CLI image: ${pact_cli_image}"
```

Please update the other variations also.